tests/testthat/alignment/fun-decs-out.R

```diff
@@ -1,38 +1,27 @@
 # aligned
-function(x   = NULL,
-         tt  = NULL,
-         ayz = NULL) {}
-
+function(x = NULL, tt = NULL, ayz = NULL) {
```

Comment:
The empty braces will be fixes as part of https://github.com/posit-dev/air/issues/186